README.md

```diff
@@ -127,9 +129,29 @@ You can overwrite the colors using css variables, for example:
   --text-highlight: #ffbb00;
 }
 ```
+### Custom Content
+
+To overwrite content (f.e. header title, home page, etc) you just need to define the global variable `UIJarConfiguration`, please note that it should be created before angular bootstraps.
+
+One way of doing it is to add `<script>` tag in your `index.html`, like the following example:
+
+```html
+<script>
+    window.UIJarConfiguration = {
+        title: `My custom title`,
+        homePage: `
```

Comment:
I guess homePage should be homeContent? :)

Reply:
Ops :p